dm/dm/proto/dmworker.proto

```diff
@@ -146,6 +146,8 @@ message SyncStatus {
     bool synced = 10;  // whether sync is catched-up in this moment
     string binlogType = 11;
     int64 secondsBehindMaster = 12; // sync unit delay seconds behind master.
+    string blockDDLOwner = 13; // block ddl upstream info, format "task-`upstreamDatabase`.`upstreamTable`"
```

Comment:
also need to add new fields in openapi [SycnStatus](https://github.com/pingcap/tiflow/blob/e40d1b8ead22cd078508f56432dc2c919310198b/dm/openapi/spec/dm.yaml#L1197)
can do this later